airflow/models/baseoperator.py

```diff
@@ -566,7 +566,20 @@ def priority_weight_total(self):
     @cached_property
     def operator_extra_link_dict(self):
         """Returns dictionary of all extra links for the operator"""
-        return {link.name: link for link in self.operator_extra_links}
+        from airflow.plugins_manager import operator_extra_links
+
+        op_extra_links_from_plugin = {}
+        for ope in operator_extra_links:
+            if ope.operators and self.__class__ in ope.operators:
```

Comment:
```suggestion
            if type(self) in ope.operators:
```

Is enough now I think? (and switch to `type(self)` as it avoids accessing `__` vars.

Reply:
Agree

Reply:
Actually pylint complains with:

```
Using type() instead of isinstance() for a typecheck.
```

Reply:
Bah, stupid pylint.